File: C++/Anfänger/klausur.cpp
#include <iostream>
using namespace std;

int Sum(int n)
{
	 int s=0;
	for (int i=0; i<=n; i++)
	{
		s+=i;
	}
	return s;
}

int main()
{
	cout << "Bitte Zahl eingeben: ";
	int n;
	cin >> n;
	int summe = Sum(n);
	cout << "Summe von 1 bis " << n << " ist: " << summe << endl;
}

File: Python/makini/Python/klausur.py
 
def sum(n):
	s = 0
	for i in range(n):
		s += i+1
	return s
	
def main ():
	print ("Zahl: ")
	n = input()
	summe=sum(n)
	print " Summe von 1 bis %f ist: %f" %(n, summe)
	
	
if __name__ == '__main__':
	main()
